hack/test-cmd.sh

```diff
@@ -126,7 +126,7 @@ hyper_storage_drivers=(
 )
 
 hyper_exec_drivers=(
-  "kvm"
+  ""
 )
```

Comment:
driverloader will decide the hypervisor. In the trivals environment, only qemu is available, so actually we test both driverloader and qemu driver.